website/index.html

```diff
@@ -16,9 +16,15 @@
         <table id="results-table" class="hidden"></table>
         <label>
             Profile URL
-            <input id="text-input" type="search" placeholder="https://codingame.com/profile/123">
-            <button id="submit-button">Add</button>
+            <input id="profile-text-input" type="search" placeholder="https://codingame.com/profile/123">
+            <button id="profile-submit-button">Add</button>
         </label>
+        <label>
+            Usernames
+            <input id="usernames-text-input" type="search" placeholder="username1,username2">
+            <button id="usernames-submit-button">Add</button>
+        </label>
+    </div>
```

Comment:
Might as well use real usernames :slightly_smiling_face: 

Probably a good idea to add a note about CodinGame not being supported for the username search?

```suggestion
        <label>
            Username*
            <input id="usernames-text-input" type="search" placeholder="andriamanitra,daxida">
            <button id="usernames-submit-button">Add</button>
        </label>
        <small>* Search by username is not supported for CodinGame</small>
```

Reply:
Sure, I thought `username1,username2` was clearer on the intent but why not.

I'm not convinced on the singular of `Username` though.

Reply:
> Did you remove a </div> or is the diff just odd?

I removed it, there's no `<div>` to close!